services/queue/config.yml

```diff
@@ -137,7 +140,8 @@ test:
       parallelism:                1
     dependencyResolver:
       pollingDelay:               250
-    taskCacheMaxSize: 10
+    taskCacheMaxSize:             10
+    taskMaxDependencies:          100
```

Comment:
Here we have `taskMaxDependencies: 100` but below in `services/queue/schemas/constants.yml` we have `max-task-dependencies: 10000` and in `services/queue/src/main.js` we also have `DEFAULT_MAX_TASK_DEPENDENCIES = 10000`. What is the reason for there being three constant defaults defined, and why are they different?

Reply:
This is a `test:` section of the config and defines variables that will be used in tests only. Also a way to test if overrides are working. 
We could also remove it and let the default take place